scripts/smoke_tests/setup.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+import sys
+import json
+import time
+import threading
+
+import requests
+
+from tests.functional.backend.common import BaseFunctionalTestCase
+
+# min amount of published collections expected by smoke tests
+MIN_TEST_COLLECTIONS = 4
+# amount advised by Timmy Huang to reduce flakiness
```

Comment:
Do you know why 10 helps reduce flakiness? If so, maybe add a little more insight here?